examples/aws/s3/package.json

```
@@ -15,10 +15,12 @@
         "ts-loader": "^9.4.1",
         "typescript": "^4.8.4",
         "webpack": "^5.74.0",
-        "webpack-cli": "^4.10.0"
+        "webpack-cli": "^4.10.0",
+        "@fermyon/knitwit": "0.3.0"
     },
     "dependencies": {
-        "@aws-sdk/client-s3": "^3.600.0",
-        "@fermyon/spin-sdk": "^2.0.0"
+        "@fermyon/spin-sdk": "^3.0.0-rc1",
```

Here and everywhere else: let's go with `3.0.0` directly.
```suggestion
        "@fermyon/spin-sdk": "^3.0.0",
```